adafruit_lps2x.py

```diff
@@ -74,7 +81,9 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(
+        cls, value_tuples: Iterable[Tuple[str, int, Optional[float], Optional[float]]]
```

Comment:
I'm torn on what the third argument in the tuple should be.  Part of me thinks that it should be on a driver-by-driver basis, and so here it should be `int`.  On the other hand, it might just make sense to type it as `Any` and just make anything wanting to use it check the type as needed.  I think the `lsb` equivalent is fine enough to pin down to `Optional[float]` across the libraries, but this argument varies more...

I think overall there's an argument to be made about the first, but perhaps we can go back and fix all the CV classes across the drivers at a later point in time.  Maybe it's worth it's own enum-like helper driver at that point.  But for now, let's be specific to it's use case here and type it as `int`.

Reply:
the reason I typed it float is because LPS25 adds 12.5 to it....so since float encompasses int, I used it.
 ("LPS25_RATE_12_5_HZ", 3, 12.5, None)

Reply:
and yes, as many different CV classes as I've seen in these drivers, some with three, some with four and some with five arguments, I've wondered to myself how, when using multiple sensors , it all gets kept straight.....

Reply:
Ah, thank you, I missed the 12.5!